apps/ios/Shared/Views/Chat/Group/GroupChatInfoView.swift

```diff
@@ -259,6 +261,13 @@ struct GroupChatInfoView: View {
     }
 }
 
+func cantInviteIncognitoAlert() -> Alert {
+    Alert(
+        title: Text("Can't invite contacts!"),
+        message: Text("You're using an incognito profile for the group - inviting your contacts may expose your main profile or other incognito profiles you've shared with them")
```

Comment:
```suggestion
        message: Text("You're using an incognito profile for this group - to prevent sharing your main profile inviting contacts is not allowed.")
```